home/templates/home/user_page.html

```diff
@@ -0,0 +1,26 @@
+{% extends "home/base.html" %}
+{% load static %}
+{% block content %}
+<link rel="stylesheet" type="text/css" href="{% static 'home/CSS/user_page.css' %}">
+<body class="main-bg">
+<main role="main" class="container">
+ <h1 class="title">{{ profile }} Profile page</h1>
```

Comment:
If the profile name is "Lior", we get from this line: "Lior Profile page".
The reason of that is because __str__ is returning `'{self.user.username}'.format(self=self)`.
The problem is that if `__str__` will be changed in the future, we will get a different output here.
Therefore, I think it should be {{ profile.user.username }} instead of {{ profile }}.

Reply:
great comment